package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "gulp docs-generate && webpack-dev-server --inline --hot --content-base src/docs/",
     "docs-upload-gh": "scripts/docs-upload-gh.sh",
     "docs-upload": "gulp docs-upload",
-    "jest": "NODE_ENV=test TZ=UTC jest",
+    "jest": "NODE_ENV=test TZ=UTC jest --runInBand",
```

Comment:
Totally, we usually use the `npm run watch` command to run tests while developing and that isn't affected by this, so it's a workable way to use multiple threads.